src/fix_langevin.cpp

```diff
@@ -105,8 +111,11 @@ FixLangevin::FixLangevin(LAMMPS *lmp, int narg, char **arg) :
       iarg += 2;
     } else if (strcmp(arg[iarg],"gjf") == 0) {
       if (iarg+2 > narg) error->all(FLERR,"Illegal fix langevin command");
-      if (strcmp(arg[iarg+1],"no") == 0) gjfflag = 0;
-      else if (strcmp(arg[iarg+1],"yes") == 0) gjfflag = 1;
+      if (strcmp(arg[iarg+1],"no") == 0) {gjfflag = 0; osflag = 0;}
+      else if (strcmp(arg[iarg+1],"yes") == 0)
+        error->all(FLERR,"Fix langevin gjf yes is outdated, please use vhalf or vfull");
```

Comment:
I think this special error message is unnecessary.  Anything other than vfull or vhalf should give the standard "Illegal fix langevin command" message.  Also, it is a good idea to consistently refer to vfull and vhalf in that order in the code, since that is the order in the documentation. You should also try to use standard LAMMPS whitespace style:
else if (strcmp(arg[iarg+1],"vhalf") == 0) {
   gjfflag = 1;
   osflag = 1;
}

This is not yet codified anywhere, but we know it when we (don't) see it.